### PR TITLE
Fix the other 2 python scripts that generates license.

### PR DESCRIPTION
**Pull request by didip**
### Description

Fixes YAML.load_all issues on two of the Python scripts that generate license.

The broken Python files interfere with some of the Maven tasks.

<hr>

##### Key changed/added classes in this PR
 * `generate-binary-notice.py`
 * `generate-binary-license.py`

<hr>

This PR has:
- [x] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [x] been tested in a test Druid cluster.

### CODE REVIEW

File: distribution/bin/generate-binary-notice.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def generate_notice(source_notice, dependences_yaml):
     # Print Apache license first.
     print_outfile(source_notice)
     with open(dependences_yaml, encoding='utf-8') as registry_file:
-        dependencies = list(yaml.load_all(registry_file))
+        dependencies = list(yaml.load_all(registry_file, Loader=yaml.Loader))
```
Comment (cryptoe) [inline distribution/bin/generate-binary-notice.py:60]: Can you check your YAML version ?
Is this working with pyyaml==5.4.1 and pyyaml==6.0.0 ?

Comment (didip) [reply distribution/bin/generate-binary-notice.py:60]: @cryptoe Both pyyaml==5.4.1 and pyyaml==6.0.0 works on my hello world test.

```python
import yaml
print(yaml.load_all(open('hello.yaml', 'r'), Loader=yaml.Loader))
```

File: pom.xml [hunk 2]
```diff
@@ -1835,6 +1835,8 @@
                                 <exclude>.editorconfig</exclude>
                                 <exclude>**/hadoop.indexer.libs.version</exclude>
                                 <exclude>**/codegen/**</exclude>
+                                <exclude>website/target/**</exclude>
```
Comment (cryptoe) [inline pom.xml:1838]: IMHO These exclusions should not be required.

File: distribution/bin/generate-binary-license.py [hunk 3]
```diff
@@ -102,7 +102,7 @@ def print_license(license):
                     license_phrase += "see {}".format(each_file)
                 else:
                     license_phrase += ", {}".format(each_file)
-
```
Comment (cryptoe) [inline distribution/bin/generate-binary-license.py:105]: nit these changes looks accidental.

File: distribution/bin/generate-binary-license.py [hunk 4]
```diff
@@ -133,11 +133,11 @@ def print_license_name_underbar(license_name):
 
 def generate_license(apache_license_v2, license_yaml):
     print_log_to_stderr("=== Generating the contents of LICENSE.BINARY file ===\n")
-
```
Comment (cryptoe) [inline distribution/bin/generate-binary-license.py:136]: nit these changes looks accidental.

File: pom.xml (hunk 2 above)
Comment (didip) [reply pom.xml:1838]: @cryptoe This is not caused by my changes, but if that exclusion is not included, Travis will always fail during this maven task.

`target/` folder is always auto generated code, yes? Isn't it ok to exclude?

File: distribution/bin/generate-binary-license.py (hunk 3 above)
Comment (didip) [reply distribution/bin/generate-binary-license.py:105]: @cryptoe That's just my editor auto removing trailing whitespace.

File: distribution/bin/generate-binary-license.py (hunk 4 above)
Comment (didip) [reply distribution/bin/generate-binary-license.py:136]: @cryptoe That's just my editor removing trailing whitespace.

File: distribution/bin/generate-binary-license.py (hunk 3 above)
Comment (cryptoe) [reply distribution/bin/generate-binary-license.py:105]: We generally try to avoid such changes.

File: pom.xml (hunk 2 above)
Comment (cryptoe) [reply pom.xml:1838]: As your py changes are not related to pom.xml, IMHO these changes are not required. 
If you think stuff is already broken in the master, please feel free to raise a separate PR.

File: distribution/bin/generate-binary-license.py (hunk 4 above)
Comment (cryptoe) [reply distribution/bin/generate-binary-license.py:136]: We generally try to avoid such changes.

File: pom.xml (hunk 2 above)
Comment (didip) [reply pom.xml:1838]: @cryptoe I can revert this.

Comment (didip) [reply pom.xml]: Reverted. Raised this instead: https://github.com/apache/druid/pull/12358

**Review by cryptoe — APPROVED** [review]
These changes LGTM. 
Thanks, @didip for the contribution.

**Review by abhishekagarwal87 — APPROVED** [review]

**Comment by abhishekagarwal87** [comment]
Thank you @didip for your contribution.

**Pull request merged**